src/cryptonote_basic/cryptonote_basic.h

```diff
@@ -77,15 +77,15 @@ namespace cryptonote
   // outputs <= HF_VERSION_VIEW_TAGS
   struct txout_to_key
   {
-    txout_to_key() { }
+    txout_to_key() = default;
```

Comment:
From cppreference.com under "implicitly-defined default constructor":

> it has the same effect as a user-defined constructor with empty body and empty initializer list.

So this should be the same as existing code? Or is that the intent?

Reply:
Yes it should... I looked into it a bit more and I'm convinced that there's a bug with my version of g++ or maybe there's some obscure undefined behavior that is triggered in the `verRctNonSemanticsSimple.serializable_sig_changes` unit test. Either way, changing a `._push_back({})` to a `._emplace_back()` fixed the warning which was generating pages and pages of garbage messages.